setup.py

```diff
@@ -7,6 +7,11 @@
 
 from setuptools import setup, find_packages
 
+# Make the current nilearn sources importable.
```

Comment:
> I would rather that you use the mechanism used below in "get_version", that doesn't rely on importing nilearn

+1. I reckon with your changes `python setup.py install` won't work if nibabel is not installed. This is because you are importing nilearn (and hence executing `nilearn/__init__.py`) which requires nibabel before giving a chance to setup to actually install nibabel. If I am not clear enough, there is a better explanation [there](https://github.com/audreyr/cookiecutter-pypackage/issues/1). This is where the exec trick is coming from by the way.